share/spack/gitlab/cloud_pipelines/stacks/tutorial/spack.yaml

```diff
@@ -1,6 +1,7 @@
 spack:
   view: false
   concretization: separately
+  deprecated: true
```

Comment:
```suggestion
  config:
    deprecated: true
```

Reply:
@tgamblin We need to add `deprecated` under the config section that is already in the `spack.yaml`